example_test.go

```diff
@@ -2,10 +2,9 @@ package ssh_config_test
 
 import (
 	"fmt"
+	"github.com/kevinburke/ssh_config"
```

Comment:
This import should stay where it was

Reply:
the imports got sorted due to an automatic gofmt run after a file save,
I do not unterstand why `github.com/kevinburke/ssh_config` needs to be at the and of the import block here.

Is it okay for you to explicitly put the import line at the end; like thiss ... : 


```
import (
	"fmt"
	"path/filepath"
	"strings"
)

import "github.com/kevinburke/ssh_config"
```